core/frontend/src/tile/IDBTileStorage.ts

```diff
@@ -0,0 +1,182 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Tiles
+ */
+
```

Comment:
Can you create an interface e.g. `IStorageCache` with a virtual function `fetch` and make IDBStorage inherit from it?

Then, as step too, can you create another class `IDBPassThrowCache`, which just recalls fetch? We will need both classes in the future.